messaging/EAP7-669_artemis_clusters_with_http_load_balancer.adoc

```diff
@@ -0,0 +1,65 @@
+= Support messaging clusters (Artemis Clusters) behind http load balancers
+:author:            Emmanuel Hugonnet
+:email:             ehugonne.email@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          messaging,jms,load-balancer
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-669
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ENTMQBR-1001
+* https://issues.apache.org/jira/browse/ARTEMIS-1322
+* https://issues.apache.org/jira/browse/EAP7-704
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+== Requirements
+
+* Add a boolean `use-topology-load-balancing` attribute to 
+- the https://wildscribe.github.io/WildFly/16.0/subsystem/messaging-activemq/connection-factory/[/subsystem=messaging-activemq/connection-factory resource]
+- the https://wildscribe.github.io/WildFly/16.0/subsystem/messaging-activemq/server/connection-factory[/subsystem=messaging-activemq/server=*/connection-factory resource]
+- the https://wildscribe.github.io/WildFly/16.0/subsystem/messaging-activemq/pooled-connection-factory/[/subsystem=messaging-activemq/pooled-connection-factory resource]
+- the https://wildscribe.github.io/WildFly/16.0/subsystem/messaging-activemq/server/pooled-connection-factory/[/subsystem=messaging-activemq/server=*/pooled-connection-factory resource]
+
```

Comment:
What are the default values of these attributes?

Reply:
fixed